kivy/tests/test_window_base.py

```diff
@@ -126,3 +127,19 @@ def test_window_opacity_clamping_negative(self):
         if self.check_opacity_support():
             self.Window.opacity = -1.5
             self.assertEqual(self.Window.opacity, 0.0)
+
+
+class WindowNativeHandleTest(GraphicUnitTest):
+
+    def test_native_handle_non_zero(self):
+        win = self.Window
+        native_handle = win.native_handle
+
+        if setupconfig.USE_X11:
+            self.assertNotEqual(native_handle, 0)
+
+        if setupconfig.USE_WAYLAND:
+            self.assertNotEqual(native_handle, 0)
+
+        if setupconfig.PLATFORM == 'win32':
```

Comment:
This check is different from the one in the method being tested, but this is what is used in `test_window_info.py` so I figured maybe your CI expects this exact configuration.